commands/group.js

```diff
@@ -80,14 +80,32 @@ module.exports = {
 			if(!args[2]) return `No ${cfg.lang} name given.`;
 			group = await bot.db.groups.get(msg.author.id, args[1]);
 			if(!group) return "You don't have a group with that name.";
-			if(args[2] == "*") {
-				await bot.db.groups.removeMembers(group.id);
-				return "All members removed from the group.";
+			args = args.slice(2);
+
+			if (args.length == 1) {
+				tup = await bot.db.members.get(msg.author.id, args[0]);
+				if(!tup) return "You don't have a registered " + cfg.lang + " with that name.";
+				await bot.db.members.removeGroup(tup.id);
+				return `${proper(cfg.lang)} '${tup.name}' group unset.`;
 			}
-			tup = await bot.db.members.get(msg.author.id, args.slice(2).join(" "));
-			if(!tup) return "You don't have a registered " + cfg.lang + " with that name.";
-			await bot.db.members.removeGroup(tup.id);
-			return `${proper(cfg.lang)} '${tup.name}' group unset.`;
+
+			let removedMessage = `${proper(cfg.lang)}s removed from group:`
+			let notRemovedMessage = `${proper(cfg.lang)}s not found:`
+			let rBaseLength = 2000 - (removedMessage.length + notRemovedMessage.length)
+			let rOriginalLength = { removedMessage: removedMessage.length, notRemovedMessage: notRemovedMessage.length, }
+
+			for await (let arg of args) {
+				tup = await bot.db.members.get(msg.author.id, arg);
+				if (tup) {
+					await bot.db.members.removeGroup(tup.id);
```

Comment:
this should be done in one query

Reply:
How? I still need to know if the tupper exists or not.

Reply:
`update` queries return the number of rows affected, so using a `where user_id = $1 and name in (unnest stuff)` will automatically update only those tuppers which exist and then return the number that did exist and were updated